data/yam/autoyast/support_images/sles12sp5_install_all_patterns_s390x.xml

```diff
@@ -35,14 +35,20 @@
     </mode>
     <final_reboot config:type="boolean">true</final_reboot>
   </general>
-  <networking>
   <firewall>
+    <FW_CONFIGURATIONS_DMZ>sshd vnc-httpd vnc-server</FW_CONFIGURATIONS_DMZ>
```

Comment:
Is `vnc-httpd` required for our openQA tests?

Reply:
Others LGTM :)

Reply:
Not required, I can remove it. :)